src/NzbDrone.Core/Notifications/PushBullet/PushBulletSettings.cs

```diff
@@ -28,10 +28,10 @@ public PushBulletSettings()
         [FieldDefinition(0, Label = "Access Token", Privacy = PrivacyLevel.ApiKey, HelpLink = "https://www.pushbullet.com/#settings/account")]
         public string ApiKey { get; set; }
 
-        [FieldDefinition(1, Label = "Device IDs", HelpText = "List of device IDs (leave blank to send to all devices)", Type = FieldType.Device)]
+        [FieldDefinition(1, Label = "Device IDs", HelpText = "List of device IDs (leave blank to send to all devices)", Type = FieldType.Device, Placeholder = "123456789,987654321")]
         public IEnumerable<string> DeviceIds { get; set; }
 
-        [FieldDefinition(2, Label = "Channel Tags", HelpText = "List of Channel Tags to send notifications to", Type = FieldType.Tag)]
+        [FieldDefinition(2, Label = "Channel Tags", HelpText = "List of Channel Tags to send notifications to", Type = FieldType.Tag, Placeholder = "Channel1234,Channel4321")]
```

Comment:
Do we need placeholder on these two? Doesn't hurt anything, but looks a tad messier.

Reply:
No need for it, I just thought it may give the user an idea of what to put in for multi values